tests/test_sessions.py

```diff
@@ -0,0 +1,92 @@
+
+import bayeslite
+import bdbcontrib
+from bdbcontrib import sessions
+
+# TODO MAKE INTO AN ACTUAL TEST
```

Comment:
Oh, I see. Yes, please do make this into an automatic test, rather than a manual one?